src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_FareContractsScreen.tsx

```diff
@@ -224,6 +224,9 @@ const useStyles = StyleSheet.createThemeHook((theme: Theme) => ({
   content: {
     padding: theme.spacings.medium,
   },
+  fareContract: {
+    paddingBottom: theme.spacings.medium,
+  },
```

Comment:
Not in use?

Reply:
Removed this and removed the style on SimpleFareContract 😄